tests/bug/gh-703.hoon

```diff
@@ -0,0 +1,14 @@
+::  list addressing bugs: &33:~ bails out
+::
+/+  *test
+|%
+::
+++  test-hang
+  %-  expect-fail
+    |.  &33:~[[%leaf p="syntax error"] [%leaf p="\{1 11}"]]
+::
+++  test-vere-bail
+  ~&  [`path`% running-unsafe+"bails out with out of loom error"]
```

Comment:
```
`path`%
```

this is rather ugly; i assume this is to put the path into the stack trace for when vere itself bails, but if vere bails, the sigpam is unlikely to trigger, no?

Reply:
That is ugly -- I meant to remove it. Already removed from the "hang" test case.